util/env/env.go

```diff
@@ -126,9 +126,17 @@ func StringFromEnv(env string, defaultValue string) string {
 	return defaultValue
 }
 
+// StringsFromEnv parses given value from the environment as a list of strings,
+// using seperator as the delimeter, and returns them as a slice. The strings
+// in the returned slice will have leading and trailing white space removed.
 func StringsFromEnv(env string, defaultValue []string, separator string) []string {
 	if str := os.Getenv(env); str != "" {
-		return strings.Split(str, separator)
+		ss := strings.Split(str, separator)
+		ret := make([]string, len(ss))
+		for i, s := range strings.Split(str, separator) {
+			ret[i] = strings.TrimSpace(s)
```

Comment:
couldn't we just do `ss[i] = strings.TrimSpace(s)` and not make a copy of the array? Not that we have to worry about performance here, but just seems unnecessary 😄

Reply:
Yep, good call.